src/commands/publish/route.rs

```diff
@@ -29,7 +28,7 @@ pub fn publish_routes(
 }
 
 fn fetch_all(user: &GlobalUser, zone_identifier: &str) -> Result<Vec<Route>, failure::Error> {
-    let client = cf_v4_api_client(user, HttpApiClientConfig::default())?;
```

Comment:
`HttpApiClientConfig::default()` <- what does this eval to?

Reply:
https://github.com/cloudflare/cloudflare-rs/blob/master/cloudflare/src/framework/mod.rs#L69